### ISSUE TITLE
Version of Windows

### ISSUE THREAD

**Issue by jimmy-1000** [Baldomo/open-in-mpv#19]
### Question Description

Does last version 2.1.0 work with Windows 7 ?
In my case doesn't work, I use Windows 7 64 bits. 

### Checklist:

- [X] I have checked for existing issues that describe my questions prior to opening this one.
- [X] I understand that improperly formatted questions may be closed without explanation.

**Comment by Baldomo** [comment]
Thank you for the report but please include more details on your setup and what you get running the application (be it errors or whatever).

**Comment by jimmy-1000** [comment]
> Thank you for the report but please include more details on your setup and what you get running the application (be it errors or whatever).
> In general, Go still supports Windows 7 so everything should work.

I don't have any setup. I don't use config.yml because this never worked for me.  When I click in "Open this link in mpv" the video doesn't open and Windows OS doesn't tell me nothing, no errors, no black window, no black mpv window, nothing. I went back to the previous 2.0.1.

**Comment by Baldomo** [comment]
Does version 2.0.1 work on Windows 7 while 2.1.0 doesn't? Is that what you mean?

**Comment by jimmy-1000** [comment]
> Does version 2.0.1 work on Windows 7 while 2.1.0 doesn't? Is that what you mean?

Yes. I have been using 2.0.1 for a long time. I works good except for this https://github.com/Baldomo/open-in-mpv/issues/9

**Comment by Tremolo4** [comment]
I have the exact same issue. Only found this project today. Version 2.0.1 works,

**Comment by Tremolo4** [comment]
Still applies to current build 2.2.1. Windows says "This app can't run on your PC". Opening it in x64dbg says "Invalid PE file". Could something be wrong with the Windows builds?

**Comment by Baldomo** [comment]
The latest release should fix this and other Windows incompatibility problems. Please test it and let me know.

**Comment by jimmy-1000** [comment]
> The latest release should fix this and other Windows incompatibility problems. Please test it and let me know.

Sadly it doesn't work.
